src/components/addBook.js

```diff
@@ -29,7 +30,8 @@ const AddBook = () => {
         <option value="Fiction">Fiction</option>
         <option value="Romance">Romance</option>
       </select>
-      <input type="submit" value="Add Book" />
+      <input type="name" placeholder="Author" />
+      <button type="submit">Add Book</button>
```

Comment:
- [ ] As one of the goals of the project is to 'Understand the concept of state management.' and according to the requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/231465718-516a6303-7f37-4ca9-884b-961bcf7cbe78.png)

Please use an event handler on the button rather than on the form, you would not have to do e.preventDefault() and also will have to manage the state of your inputs like `const [title, setTitle] = useState("")` these are important skills to learn, even though currently the output would be same the Implementation is what we are more interested in :+1:

Reply:
Thanks for pointing that out.

Reply:
fixed!